modules/oadp-ceph-split.adoc

```diff
@@ -0,0 +1,165 @@
+
+// [IMPORTANT]
+// ====
+// The CephFS shallow copy feature can only be used for Data Mover backup operations.  The shallow copy volume options are not supported for restore.
+// ====
+
+.Prerequisites
+
+* Ensure a stateful application is running in a separate namespace with PVCs provisioned by both CephFS and CephRBD.
+
+* CephFS is the default `StorageClass` and `VolumeSnapshotClass`.
+
+Procedure
+
+. Create a Data Protection Application (DPA) CR similar to one below. Use the Restic `Secret` you created when you prepared your environment for working with OADP 1.2 Data Mover and Ceph as the value for `spec.features.dataMover.credentialName`. If you do not, then the CR will use the default value `dm-credential` for this parameter.
++
+[NOTE]
+====
+`VolumeOptionsForStorageClass` can be defined for multiple `storageClasses`, thus allowing a backup to complete with volumes with different providers.
+====
++
+Example DPA CR for environment with split volumes
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
+          prefix: velero
+        provider: aws
+  configuration:
+    restic:
+      enable: false
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+        - vsm
+  features:
+    dataMover:
+      credentialName: <restic-secret-name>
+      enable: true
+      volumeOptionsForStorageClasses:
+        ocs-storagecluster-cephfs:
+          sourceVolumeOptions:
+            accessMode: ReadOnlyMany
+            cacheAccessMode: ReadWriteMany
+            cacheStorageClassName: ocs-storagecluster-cephfs
+            storageClassName: ocs-storagecluster-cephfs-shallow
+        ocs-storagecluster-ceph-rbd:
+          sourceVolumeOptions:
+            storageClassName: ocs-storagecluster-ceph-rbd
+            cacheStorageClassName: ocs-storagecluster-ceph-rbd
+        destinationVolumeOptions:
+            storageClassName: ocs-storagecluster-ceph-rbd
+            cacheStorageClassName: ocs-storagecluster-ceph-rbd
+----
+
+. To back up data:
+
+.. Create a `Backup` CR:
++
+Example `Backup` CR
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns>
+spec:
+  includedNamespaces:
+- <app-ns>
+storageLocation: velero-sample-1
+----
+
+.. Monitor the Data Mover backup and its artifacts by running the script named link:https://github.com/openshift/oadp-operator/blob/master/docs/examples/datamover_resources.sh[Volumemover live debug]
```

Comment:
Script again :) Stopping with comments here on this file - basically the same as for the other version above. I do wonder if you break these into separate procedures if you can re-use them so that you don't have quite so many files to juggle :thinking: